Documentation/PlatformSupport.md

```diff
@@ -0,0 +1,69 @@
+
+# Platform Support
+
+Darwin platforms are binary and source stable. Linux platforms follow semantic versioning. Windows follows semantic versioning.
+
+### Windows concerns
+
+Trying to perfectly unify Windows with Darwin and Linux at the API level is ultimately a futile endeavor. However, in some specific cases, Windows does have directly corresponding syscalls or near-equivalents.
+
+Windows-specific details are open to reevaluation pending more Windows expertise for the project.
+
+
+## API to syscall mapping
+
+### `Errno`
+
+#### Instance computed properties
+
+| API                | Darwin     | Linux      | Windows    |
+|:-------------------|:-----------|:-----------|:-----------|
+| `description`      | `strerror` | `strerror` | `strerror` |
+| `debugDescription` | `strerror` | `strerror` | `strerror` |
```

Comment:
Is this meant to cover the current state or the desired state?  I think that for Windows, we should consider either `_strerror_s` or `_wstrerror_s`.  The downside is that it requires a duplicated call, one to size the buffer which the user allocates and the call to convert the string.

Reply:
This covers the current state, where we will make a `String` copy from the pointer given to us. This is not the most performance-critical path. What's the advantage of those calls?

Reply:
The advantage of the calls is avoiding the annoying warnings about deprecation.  Microsoft has long been trying to push for users to migrate to APIs that are user-allocated buffers rather than static buffers and to calls that take buffer pointer and size rather than buffer pointer only.  These are supposed to be more "secure" as a result (at least should help alleviate some of the buffer overruns).

Reply:
Ok, it doesn't seem as critical in that case since we immediately copy the contents into a new String and do not expose the static buffer pointer. Please open an issue if this is something we should still track.